<!DOCTYPE html
  PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
  "DTD/xhtml1-strict.dtd">
<html>
  <head>
    <title>API docs for &ldquo;twisted.spread.pb.Error&rdquo;</title>
    <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
    <link href="apidocs.css" type="text/css" rel="stylesheet" />
    
    
  </head>
  <body>
    <h1 class="class">Class t.s.p.Error(<span title="Exception">Exception</span>):</h1>
    <p>
      <span id="part">Part of <a href="twisted.html">twisted</a>.<a href="twisted.spread.html">spread</a>.<a href="twisted.spread.pb.html">pb</a></span>
      <a href="http://twistedmatrix.com/trac/browser/tags/releases/twisted-9.0.0/twisted/spread/pb.py#L91">View Source</a>
      <a href="classIndex.html#twisted.spread.pb.Error">View In Hierarchy</a>
    </p>
    <div>
      
    </div>
    <div>This error can be raised to generate known error conditions.</p>
<p>When a PB callable method (perspective_, remote_, view_) raises this 
error, it indicates that a traceback should not be printed, but instead, 
the string representation of the exception should be sent.<table class="fieldTable"></table></div>

    
    
    <div id="splitTables">
      
      
      
    </div>
    
    
    

    
    <address>
      <a href="index.html">API Documentation</a> for <a href="http://twistedmatrix.com/">Twisted</a>, generated by <a href="http://codespeak.net/~mwh/pydoctor/">pydoctor</a> at 2009-11-24 22:07:21.
    </address>

    <!-- Google analytics, obviously. -->
    <script src="http://www.google-analytics.com/urchin.js" type="text/javascript"></script>
    <script type="text/javascript">
      _uacct = "UA-99018-6";
      urchinTracker();
    </script>
  </body>
</html>